cmd/sandbox/sandbox_util.go

```diff
@@ -87,34 +90,44 @@ func startContainer(cli *client.Client) (string, error) {
 	if err != nil {
 		return "", err
 	}
-
-	if _, err := io.Copy(os.Stdout, r); err != nil {
-		return "", err
+	if sandboxConfig.DefaultConfig.Debug {
```

Comment:
pass the debug flag as parameter to the function and have the caller pass it from one place.

Reply:
done